webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrengthSelectors.js

```diff
@@ -1,2 +1,6 @@
 export const doesPasswordsMatch = ({ password, passwordConfirmation }) =>
   !passwordConfirmation || password === passwordConfirmation;
+
+export const passwordPresent = passwordStrength => (
+  passwordStrength ? passwordStrength.password : undefined
```

Comment:
afik `passwordStrength` initial state exists with empty pasword: 
``` javascript
{
  password: '',
  passwordConfirmation: '',
}
```
therefore this might be redundant 
anyway, instead of this if statement you can use `&&` opertaor:
``` javascript 
passwordStrength && passwordStrength.password
```